js/templates/modals/baseModal.html

```diff
@@ -1,4 +1,22 @@
+<div class="popInMessageHolder js-popInMessages"></div>
 <div class="<%= ob.modelContentClass %>">
-  <i class="jsModalClose <%= ob.closeButtonClass %> <% !ob.showCloseButton && print('hide') %>"></i>
+  <div class="topControls pureFlex spread js-closeClickTarget js-btcTicker">
```

Comment:
I personally think there is too much variance in-between what the modals are doing to abstract out the top-control piece. I think the most flexibility is having individual modals manage it themselves. If we find that we are repeating ourselves (which I don't see right now), we could look into abstracting out the common functionality. But, I certainly wouldn't put that in the baseModal. Modals are implemented in a variety of ways, for example, the BaseModal view and template are used for error / message dialogs (which are small dialogs in the center of the screen). IMO, the base modal should stay very generic.

I would recommend you do the top-control part of the About modal inside the About modal template.